### ISSUE TITLE
Pod for gloo keeps restarting

### ISSUE THREAD

**Issue by munjal-patel** [solo-io/gloo#1021]
Hello,

I installed Gloo for Knative using `glooctl install knative` on GKE.

And with that basic setup, I notice that the pod for `gloo` keeps failing and restarting.
In last 13 hours, it restarted 96 times.
Is this normal? If not, what can I do to fix this?

![image](https://user-images.githubusercontent.com/51474446/62924079-31b78200-bd64-11e9-8b1b-a518dbfc0488.png)

Here are all the pods in my cluster:
![image](https://user-images.githubusercontent.com/51474446/62924183-6e837900-bd64-11e9-8bdf-63525cae28be.png)

Helm & Tiller is configured with `tls` on.

**Comment by ilackarms** [comment]
can you run `kubectl logs -n gloo-system -l gloo=gloo`?
let's see them logs

**Comment by munjal-patel** [comment]
Sure. Here are the top 5 latest from oldest log entries at the time of crash:

```
[
 {
   "textPayload": "log: exiting because of error: log: cannot create log: open /tmp/gloo.gloo-656c7b8966-l9jps.unknownuser.log.WARNING.20190813-142044.1: read-only file system",
   "insertId": "16gw59lg3ngfgiy",
   "resource": {
     "type": "container",
     "labels": {
       "pod_id": "gloo-656c7b8966-l9jps",
       "zone": "us-west1-a",
       "project_id": "sn-289045",
       "cluster_name": "default-cluster",
       "container_name": "gloo",
       "namespace_id": "gloo-system",
       "instance_id": "2160513837961058557"
     }
   },
   "timestamp": "2019-08-13T14:20:44.411702604Z",
   "severity": "ERROR",
   "labels": {
     "container.googleapis.com/namespace_name": "gloo-system",
     "compute.googleapis.com/resource_name": "gke-default-cluster-default-pool-2e885321-5cs4",
     "container.googleapis.com/pod_name": "gloo-656c7b8966-l9jps",
     "container.googleapis.com/stream": "stderr"
   },
   "logName": "projects/sn-289045/logs/gloo",
   "receiveTimestamp": "2019-08-13T14:20:48.772469902Z"
 },
[
 {
   "textPayload": "log: exiting because of error: log: cannot create log: open /tmp/gloo.gloo-656c7b8966-l9jps.unknownuser.log.WARNING.20190813-142044.1: read-only file system",
   "insertId": "16gw59lg3ngfgiy",
   "resource": {
     "type": "container",
     "labels": {
       "pod_id": "gloo-656c7b8966-l9jps",
       "zone": "us-west1-a",
       "project_id": "sn-289045",
       "cluster_name": "default-cluster",
       "container_name": "gloo",
       "namespace_id": "gloo-system",
       "instance_id": "2160513837961058557"
     }
   },
   "timestamp": "2019-08-13T14:20:44.411702604Z",
   "severity": "ERROR",
   "labels": {
     "container.googleapis.com/namespace_name": "gloo-system",
     "compute.googleapis.com/resource_name": "gke-default-cluster-default-pool-2e885321-5cs4",
     "container.googleapis.com/pod_name": "gloo-656c7b8966-l9jps",
     "container.googleapis.com/stream": "stderr"
   },
   "logName": "projects/sn-289045/logs/gloo",
   "receiveTimestamp": "2019-08-13T14:20:48.772469902Z"
 },
 {
   "insertId": "pt70glg18k93un",
   "jsonPayload": {
     "level": "info",
     "caller": "cache/simple.go:200",
     "msg": "open watch 7 for type.googleapis.com/envoy.api.v2.RouteConfiguration[] from nodeID \"gloo-system~knative-external-proxy\", version \"0\"",
     "ts": "2019-08-13T14:12:36.851Z"
   },
   "resource": {
     "type": "container",
     "labels": {
       "project_id": "sn-289045",
       "cluster_name": "default-cluster",
       "container_name": "gloo",
       "namespace_id": "gloo-system",
       "instance_id": "2160513837961058557",
       "zone": "us-west1-a",
       "pod_id": "gloo-656c7b8966-l9jps"
     }
   },
   "timestamp": "2019-08-13T14:12:36.852042053Z",
   "severity": "ERROR",
   "labels": {
     "container.googleapis.com/pod_name": "gloo-656c7b8966-l9jps",
     "container.googleapis.com/stream": "stderr",
     "container.googleapis.com/namespace_name": "gloo-system",
     "compute.googleapis.com/resource_name": "gke-default-cluster-default-pool-2e885321-5cs4"
   },
   "logName": "projects/sn-289045/logs/gloo",
   "receiveTimestamp": "2019-08-13T14:12:42.534494802Z"
 },
 {
   "insertId": "pt70glg18k93ul",
   "jsonPayload": {
     "level": "info",
     "caller": "cache/simple.go:200",
     "msg": "open watch 5 for type.googleapis.com/envoy.api.v2.ClusterLoadAssignment[knative-serving-activator-service-activator-v0-8-0-9090_gloo-system knative-serving-autoscaler-v0-8-0-443_gloo-system knative-serving-activator-service-activator-v0-8-0-80_gloo-system knative-serving-activator-service-9090_gloo-system knative-serving-activator-service-activator-v0-8-0-81_gloo-system knative-serving-activator-service-80_gloo-system knative-serving-activator-service-81_gloo-system knative-serving-webhook-webhook-v0-8-0-443_gloo-system knative-serving-autoscaler-v0-8-0-8080_gloo-system knative-serving-autoscaler-9090_gloo-system knative-serving-autoscaler-443_gloo-system knative-serving-autoscaler-v0-8-0-9090_gloo-system knative-serving-autoscaler-8080_gloo-system knative-serving-webhook-443_gloo-system kube-svc:kube-system-tiller-deploy-44134_kube-system kube-svc:knative-serving-autoscaler-9090_knative-serving kube-svc:knative-serving-autoscaler-8080_knative-serving kube-svc:knative-serving-autoscaler-443_knative-serving kube-svc:knative-serving-activator-service-81_knative-serving kube-svc:kube-system-kube-dns-53_kube-system kube-svc:knative-serving-activator-service-9090_knative-serving kube-system-metrics-server-443_gloo-system kube-system-default-http-backend-80_gloo-system knative-serving-controller-9090_gloo-system kube-svc:knative-serving-activator-service-80_knative-serving kube-svc:gloo-system-gloo-9977_gloo-system knative-serving-controller-v0-8-0-9090_gloo-system gloo-system-knative-internal-proxy-443_gloo-system gloo-system-knative-external-proxy-443_gloo-system kube-svc:default-kubernetes-443_default kube-svc:knative-serving-controller-9090_knative-serving kube-system-default-http-backend-glbc-80_gloo-system kube-svc:knative-serving-webhook-443_knative-serving kube-svc:gloo-system-knative-internal-proxy-80_gloo-system gloo-system-knative-internal-proxy-80_gloo-system gloo-system-knative-external-proxy-80_gloo-system gloo-system-gloo-9977_gloo-system kube-svc:kube-system-heapster-80_kube-system kube-svc:kube-system-default-http-backend-80_kube-system kube-system-kube-dns-53_gloo-system kube-system-heapster-v1-6-0-beta-1-80_gloo-system kube-svc:gloo-system-knative-internal-proxy-443_gloo-system kube-svc:gloo-system-knative-external-proxy-80_gloo-system kube-svc:gloo-system-knative-external-proxy-443_gloo-system default-kubernetes-443_gloo-system kube-system-metrics-server-v0-3-1-443_gloo-system kube-svc:kube-system-metrics-server-443_kube-system kube-system-tiller-deploy-44134_gloo-system kube-system-heapster-80_gloo-system] from nodeID \"gloo-system~knative-external-proxy\", version \"468034823347115683\"",
     "ts": "2019-08-13T14:12:36.851Z"
   },
   "resource": {
     "type": "container",
     "labels": {
       "instance_id": "2160513837961058557",
       "pod_id": "gloo-656c7b8966-l9jps",
       "zone": "us-west1-a",
       "project_id": "sn-289045",
       "cluster_name": "default-cluster",
       "container_name": "gloo",
       "namespace_id": "gloo-system"
     }
   },
   "timestamp": "2019-08-13T14:12:36.851951055Z",
   "severity": "ERROR",
   "labels": {
     "container.googleapis.com/pod_name": "gloo-656c7b8966-l9jps",
     "container.googleapis.com/stream": "stderr",
     "container.googleapis.com/namespace_name": "gloo-system",
     "compute.googleapis.com/resource_name": "gke-default-cluster-default-pool-2e885321-5cs4"
   },
   "logName": "projects/sn-289045/logs/gloo",
   "receiveTimestamp": "2019-08-13T14:12:42.534494802Z"
 }
]
```

**Comment by yuval-k** [comment]
I believe this is fixed by #1022 ; a release with this will be out shortly

**Comment by yuval-k** [comment]
This issue should be fixed in the latest gloo release. Please let us know if the problem persist
